addons/addon-base/packages/services/package.json

```diff
@@ -8,7 +8,8 @@
   "dependencies": {
     "@aws-ee/base-services-container": "workspace:*",
     "ajv": "^6.11.0",
-    "aws-sdk": "^2.647.0",
+    "aws-sdk": "^2.7.15",
```

Comment:
Xray requires the aws sdk to be ^2.7.15
https://www.npmjs.com/package/aws-xray-sdk

Reply:
@nguyen102, `^2.647.0` already fulfills "v2.7.15 or greater". There's no need to change this.

Consider reverting this change. It's not breaking since npm installs the latest `2.x` version anyways, but it's a bit misleading. `2.7.15` was released 4 years ago.

Reply:
Good catch. New PR for this.
https://github.com/awslabs/service-workbench-on-aws/pull/422